src/ticketmap/index.tsx

```diff
@@ -117,11 +125,9 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
   }
 
   async fetchMap() {
-    const mapSvgUrl = `${this.configFilePath}/map.svg`
-    const mapNotAvailableUrl = 'https://maps.ticketevolution.com/maps/not_available.jpg' // Only exists on prod
-    let mapResponse = await fetch(mapSvgUrl)
+    const mapResponse = await fetch(`${this.configFilePath}/map.svg`)
     if (!mapResponse.ok) {
-      mapResponse = await fetch(mapNotAvailableUrl)
+      throw new MapNotFoundError('This map is not currently available.');
```

Comment:
The `message` for `MapNotFoundError` seems to be constant. We could set this as the default for its constructor.

```ts
class MapNotFoundError extends Error {
  constructor(message = 'This map is not currently available.') {
    super(message)
  }
}}
```